### DISCUSSION TITLE
Delete user from auth schema

### DISCUSSION THREAD

**Discussion by Jhnvlglmlbrt** [supabase/supabase#26223] [Questions] [answered]
Hi, wanted to ask - is it possible to delete user from Authentication via the code? Not using the website interface, but programmatically? I searched info and as i understand its possible to create by using diff lang. But remove?

**Comment by GaryAustin1** [comment, marked as answer]
You can delete the user row in auth.users from SQL and that will delete the user.

**Reply by Jhnvlglmlbrt** [reply]
From supabase sql editor? But not from sql queries?

**Reply by silentworks** [reply]
It should work from either. Is it not working for you? post the code you are trying here.

**Reply by Jhnvlglmlbrt** [reply]
I could do that, thanks. I did wrong queries.
